docs/release-notes/8.12.asciidoc

```diff
@@ -2,15 +2,32 @@
 == 8.12
 
 [discrete]
-[[enhancements-8.12.1]]
-==== Enhancements
+[[release-notes-8.12.2]]
+=== 8.12.2
 
-* Provides performance improvements related to image load and registry write events ({pull}175486[#175486]).
+[discrete]
+[[bug-fixes-8.12.2]]
+==== Bug fixes
+* Fixes long-running queries in Timeline and Events tables within Explore pages ({pull}176838[#176838]).
+* Updates the default {bedrock} connector API URL ({pull}176090[#176090]).
+* Ensures the risk score query only searches through alerts associated with the current user ({pull}175903[#175903]).
+* Fixes a bug that prevented numbers from being used for Osquery pack IDs ({pull}176507[#176507]).
```

Comment:
Thanks for taking care of this 🙇 
I think I explained it poorly in my PR, sorry! I would say it can be more like:
`Fixes a bug that prevents scheduled pack's query to run if query id is made of only numbers.` 
Does this make sense?

Reply:
Thanks for the additional context here, @tomsonpl! How's this:
```suggestion
* Fixes a bug that prevented scheduled query packs from running if a pack's ID was composed of numbers ({pull}176507[#176507]).
```

Reply:
Looks awesome, thank you :)